src/test/java/DemoTest.java

```diff
@@ -1,7 +1,125 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.Random;
+import java.util.concurrent.TimeUnit;
 
 public class DemoTest {
 
+  @Test
+  public void incorrectPassAndEmailTest() {
+    WebDriver driver = new ChromeDriver();
+    driver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);
+    driver.get("https://deens-master.now.sh");
+
+    driver.findElement(By.cssSelector("a[href*='login']")).click();
+
+    WebElement email = driver.findElement(By.cssSelector("#email"));
+    email.sendKeys("valery@gmail.com");
+    WebElement pass = driver.findElement(By.cssSelector("#password"));
+    pass.sendKeys("Testing123");
+    driver.findElement(By.cssSelector("[data-testid='loginSubmit']")).click();
+    WebElement errorMessage = driver.findElement(By.cssSelector(".ui.error.message p"));
+    String errorMessageText = errorMessage.getText();
+
+    Assert.assertEquals(errorMessageText, "Wrong email or password.");
+    driver.quit();
+  }
+
+  @Test
+  public void loginEmptyParametrsTest() {
+    WebDriver driver = new ChromeDriver();
+    driver.get("https://deens-master.now.sh/login");
+
+    WebElement loginButton = driver.findElement(By.cssSelector("[data-testid='loginSubmit']"));
+    loginButton.click();
+    WebElement errorMessage = driver.findElement(By.cssSelector(".ui.error.message p"));
+    String errorMessageText = errorMessage.getText();
+
+    Assert.assertEquals(errorMessageText, "Empty email or password");
+    driver.quit();
+  }
+
+  @Test
+  public void loginSuccessTest() throws InterruptedException {
+    WebDriver driver = new ChromeDriver();
+//    driver.get("https://deens-master.now.sh/login");
+    driver.get("https://deens.com/login");
+    driver.findElement(By.cssSelector("#email")).sendKeys("valery.kells0202@gmail.com");
+    driver.findElement(By.cssSelector("#password")).sendKeys("Testing12345");
+    driver.findElement(By.cssSelector(".ui.large.fluid.button.green-btn.pl-btn")).click();
+
+    Thread.sleep(6000);
```

Comment:
do not use Thread.sleep

Reply:
done

Reply:
when I'm using Thread I'm passing the test when I'm using driver.manage().timeouts().implicitlyWait(1o, TimeUnit.SECONDS) I'm failing it